src/opower/__main__.py

```diff
@@ -99,7 +99,7 @@ async def _main() -> None:
         except (FileNotFoundError, json.JSONDecodeError):
             pass
 
-    async with aiohttp.ClientSession(cookie_jar=create_cookie_jar()) as session:
+    async with aiohttp.ClientSession(cookie_jar=create_cookie_jar(), requote_redirect_url=False) as session:

```

Comment:
Can you add `allow_redirects=False` to the get or post that needs this and manually redirect?
I'd rather avoid disabling `requote_redirect_url`. We would also have to disable it in Home Assistant core where the session is created.